Boltdir/Puppetfile

```diff
@@ -4,10 +4,12 @@
 # The following directive installs modules to the managed moduledir.
 moduledir '.modules'
 
-mod 'puppetlabs/stdlib', '7.1.0'
+mod 'puppet/firewalld', '4.5.1'
```

Comment:
are you aware of ra10ke? It provides you with rake tasks that inform you when a module is outdated or deprecated

Reply:
Familiar, yes, although I have never used it. I have some [similar code](https://github.com/silug/puppet-tools/blob/master/update-puppetfile) that I use.

In this case I had to downgrade a couple of modules to make `bolt module install` happy.